puppetboard/static/js/tables.js

```diff
@@ -16,6 +16,17 @@
     else return tdTime;
   });
 
+    $('thead th.IP Address').data('myIpArray', myIpArray) // save IP adresses from th. IP Addresses (unsure about the name) in myIpArray
```

Comment:
It looks like you're trying to select `th.IP Address`, following the pattern for `.date` sort.

Are you sure this class exists on any of the table headers? If there isn't a class name identifying `IP Address` in the html, you'd have to add one for this to work.

If you do add it use an underscore `ip_address` as CSS classes can't have spaces.

Reply:
This is right clicking in chrome on a date header and clicking inspect. It shows `class="date"`.

When I do the same on anything with an IP I don't any unique class.

![untitled](https://user-images.githubusercontent.com/9503662/32498258-395ce774-c39d-11e7-9e41-a80c1caeb084.png)

Reply:
The reason I tried IP Address as a table header is when I import the headers from the table I get something like:
"Hostname,IP Address,OS,Architecture,Kernel Version,Reboot required,Puppet Version"
So IP Address must be a header so I wrote th.IP Address

Reply:
For the Inventory page I can select the `th` for IP Address with this:

    $("table").find("[aria-label^='IP Address']")

It says find in table where `aria-label` starts with 'IP Address'.

Reply:
I can see them as headers when I try the sorting on inventory. 
![ip address](https://user-images.githubusercontent.com/32731446/32499746-ba5b571c-c3d3-11e7-8d38-3b012a96b5f7.jpg)

Reply:
I see where you were coming from, but the syntax isn't correct. Just run your selector in the console tag on chrome and you will see it returns 0 items.

![untitled](https://user-images.githubusercontent.com/9503662/32499961-ff46bf7e-c3a1-11e7-91f9-ff6219b9d3c2.png)

Reply:
I might need to create a class, but then how should I identify it if only reference there is is the aria-label^='IP Address'?